src/IW/Sync/Search.hs

```diff
@@ -0,0 +1,117 @@
+module IW.Sync.Search
```

Comment:
Could we add documentation to this whole module as well so that it would be even clearer what are these endpoints and when they are going to be used? This could be useful when you looking quickly through the modules of the project.

```
{- | This module provides functions...
-}
```
What do you think?

Reply:
@vrom911 I agree! I need to add some documentation to some other modules as well.